prog/test/vm.rb

```diff
@@ -24,7 +26,7 @@ class Prog::Test::Vm < Prog::Test::Base
   label def install_packages
     if /ubuntu|debian/.match?(vm.boot_image)
       sshable.cmd("sudo apt update")
-      sshable.cmd("sudo apt install -y build-essential")
+      sshable.cmd("sudo apt install -y build-essential fio")
     elsif vm.boot_image.start_with?("almalinux")
       sshable.cmd("sudo dnf check-update || [ $? -eq 100 ]")
       sshable.cmd("sudo dnf install -y gcc gcc-c++ make")
```

Comment:
Why don't we install `fio` on `almalinux`? Does it come by default?

Reply:
No, I had forgotten. Thanks for reminding. Fixed.